go.mod

```diff
@@ -3,33 +3,33 @@ module github.com/GoogleContainerTools/kpt
 go 1.17
 
 require (
-	github.com/cpuguy83/go-md2man/v2 v2.0.0
-	github.com/go-errors/errors v1.4.0
-	github.com/google/go-cmp v0.5.6
+	github.com/cpuguy83/go-md2man/v2 v2.0.1
+	github.com/go-errors/errors v1.4.2
+	github.com/google/go-cmp v0.5.7
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/igorsobreira/titlecase v0.0.0-20140109233139-4156b5b858ac
-	github.com/otiai10/copy v1.6.0
+	github.com/otiai10/copy v1.7.0
 	github.com/philopon/go-toposort v0.0.0-20170620085441-9be86dbd762f
-	github.com/spf13/cobra v1.2.1
+	github.com/spf13/cobra v1.3.0
 	github.com/stretchr/testify v1.7.0
-	github.com/xlab/treeprint v0.0.0-20181112141820-a009c3971eca
+	github.com/xlab/treeprint v1.1.0
 	golang.org/x/mod v0.5.1
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.22.3
-	k8s.io/apiextensions-apiserver v0.22.2
-	k8s.io/apimachinery v0.22.3
-	k8s.io/cli-runtime v0.22.2
-	k8s.io/client-go v0.22.2
-	k8s.io/klog/v2 v2.10.0
+	k8s.io/api v0.23.3
+	k8s.io/apiextensions-apiserver v0.23.3
+	k8s.io/apimachinery v0.23.3
```

Comment:
This PR (https://github.com/GoogleContainerTools/kpt/pull/2796) also bumps up the k8s deps.

Reply:
Good go know. Let's merge it then, without delay; it should then reduce the diff for this merge.